ruma-events/tests/state_event.rs

```diff
@@ -236,3 +236,48 @@ fn deserialize_avatar_without_prev_content() {
             && unsigned.is_empty()
     );
 }
+
+#[test]
+fn deserialize_member_event_membership_hoist_unsigned_prev_content() {
```

Comment:
Side question (maybe unclear to me bc. English is not my first language): What does 'hoist' mean here?

Reply:
Instead of membership being in the `content` field it's at the top-level,  "lifted" out.